
map_anon_fun.erl:4:1: Function g/1 will never be called
